OngProject/Controllers/NewsController.cs

```diff
@@ -0,0 +1,177 @@
+﻿using System.Collections.Generic;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using OngProject.Core.Interfaces;
+using OngProject.DataAccess.UnitOfWork.Interfaces;
+using OngProject.Entities;
+using System;
+using System.Threading.Tasks;
+using AutoMapper;
+using OngProject.Core.Models.DTOs;
+
+namespace OngProject.Controllers
+{
+    [ApiController]
+    public class NewsController : ControllerBase
+    {
+        private readonly IUnitOfWork _uow;
+        private readonly INewsBusiness _newsBusiness;
+        private readonly IMapper _mapper;
+        public NewsController(IUnitOfWork uow, INewsBusiness newsBusiness, IMapper mapper)
+        {
+            this._uow = uow;
+            this._newsBusiness = newsBusiness;
+            this._mapper = mapper;
+        }
+
+        // GET List/News
+        [HttpGet]    
+        [Route("List/News")]
+        public async Task<IActionResult> GetAllNews() 
+        {
+            try
+            {
+                var news = _newsBusiness.FindNewsAsync(m => m.softDelete != true);
+                return news != null ? Ok(_mapper.Map<IEnumerable<NewsDTO>>(news)) 
+                                       : NotFound("The list of news has not been found");                
+            }
+            catch (System.Exception ex)
+            {
+                throw new Exception(ex.Message);
+            }           
+        }
+
+        // GET List/NewsById
+        [HttpGet]        
+        [Route("List/NewsById/{id}")]
+        public async Task<IActionResult> GetById(int id)
+        {
+            try
+            {
+                if(id == 0)
+                    return NotFound("Please, set an ID.");
+
+                var news = await _newsBusiness.GetNewsByIdAsync(id);
+                return news != null ? Ok(_mapper.Map<NewsDTO>(news)) 
+                                      : NotFound("News doesn't exists");            
+            }
+            catch (System.Exception ex)
+            {
+                throw new Exception(ex.Message);
+            }
+        }
+
+        // POST Create/News
+        [HttpPost]       
+        [Route("Create/News")]
+        public async Task<IActionResult> Create([FromBody] NewsDTO model)
+        {          
+            if(ModelState.IsValid)
+            {
+                try
+                {
+                    // validations                    
+                    if(string.IsNullOrEmpty(model.Name))
```

Comment:
Cambiaria el Ok por un status code 400, ya que interpreto que estas informando que un campo es requerido. Con un status code 400 le haces saber al cliente que envió los datos que ejecución del recurso no fue exitosa. Por otra parte, si usas DataAnnotations, que son las anotaciones [Required] en el NewsDTO por ejemplo para el campo Name seria:
[Required(ErrorMessage = "Name is required")]
public string Name { get; set; }
Al hacer usar esa anotacion, no es necesario usar el string.IsNullOrEmpty